recipes-devtools/gcc-for-nvcc/gcc-for-nvcc-10.3.inc

```diff
@@ -126,4 +126,4 @@ EXTRA_OECONF_PATHS = "\
 "
 
 # Is a binutils 2.26 issue, not gcc
-CVE_CHECK_IGNORE += "CVE-2021-37322"
+CVE_STATUS[CVE-2021-37322] = "ignored"
```

Comment:
This should specify the actual reason. Looking at the setting in OE-Core's gcc recipe, it should be `"cpe-incorrect: Is a binutils 2.26 issue, not gcc"`.

Reply:
Modified the comment

Reply:
It's the setting that needs changing, and the comment should be removed.  See corresponding line in the [current gcc recipe](https://git.yoctoproject.org/poky/tree/meta/recipes-devtools/gcc/gcc-14.2.inc#n111).

Reply:
Modified the line. Thanks for the clarification!